src/env.js

```
@@ -60,6 +60,10 @@ export const env = createEnv({
     RESEND_API_KEY: process.env.RESEND_API_KEY,
     STRIPE_SECRET_KEY: process.env.STRIPE_SECRET_KEY,
     STRIPE_WEBHOOK_SECRET: process.env.STRIPE_WEBHOOK_SECRET,
+    STRIPE_FOUNDING_MONTHLY_PRICE_ID: process.env.STRIPE_FOUNDING_MONTHLY_PRICE_ID,
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'STRIPE_FOUNDING_MONTHLY_PRICE_ID' should be before 'STRIPE_WEBHOOK_SECRET'.

(sort-keys-fix/sort-keys-fix)